Input/input_sup6.xml

```diff
@@ -5,6 +5,7 @@
   <Parameter name="RandomGeneratorSeed" type="int" value="95"/>
   <Parameter name="TopologyFilePath" type="string" value="sup2.dat"/>
   <Parameter name="MaxIteration" type="int" value="100"/>
+  <Parameter name="PressureGreenFunFlag" type="bool" value="true"/>
```

Comment:
Why is this set to true here? This changes the physics of the test case - do the tests pass with this parameter set to true?

Reply:
Currently, the tests do not compare the results with some defined values. The tests pass if the simulation runs through without any error. We can define the expected results using the results from our MATLAB code, but I am not sure how to define the results and compare them in the integration tests.

Reply:
Then one of the very next things should be to add result tests for the input files. We can have a talk on how this could be achieved. Still, please don't modify the existing test case here, but add a new one for the pressure based variant.